src/main/resources/META-INF/rust-core.xml

```diff
@@ -1009,10 +1009,22 @@
 
     </extensions>
 
+    <extensions defaultExtensionNs="org.rust">
+        <!-- SDK Flavors -->
+        <sdkFlavor implementation="org.rust.ide.sdk.flavors.RustupSdkFlavor"/>
```

Comment:
Also, I'd add `order="first"` here just to be sure that we prefer using `rustup` over all other variants. 
Note, the platform doesn't guarantee implicit order of extensions